specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2021-12-01/Marketplace.json

```diff
@@ -0,0 +1,2894 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AcknowledgeNotification": {
+            "$ref": "./examples/AcknowledgeNotification.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/privateStores/{privateStoreId}/requestApprovals/{requestApprovalId}/withdrawPlan": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/RequestApprovalIdPathParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "post": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_WithdrawPlan",
+        "description": "Withdraw a user request approval on specific plan",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "payload",
+            "schema": {
+              "$ref": "#/definitions/WithdrawProperties"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WithdrawPlan": {
+            "$ref": "./examples/WithdrawPlan.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/privateStores/fetchAllSubscriptionsInTenant": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        },
+        {
+          "$ref": "#/parameters/SkipTokenParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_FetchAllSubscriptionsInTenant",
+        "description": "Fetch all subscriptions in tenant, only for marketplace admin",
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionsResponse"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "skipToken"
+        },
+        "x-ms-examples": {
+          "FetchAllSubscriptionsInTenant": {
+            "$ref": "./examples/FetchAllSubscriptionsInTenant.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/privateStores/listNewPlansNotifications": {
```

Comment:
>listNewPlansNotifications [](http://example.com/codeflow?start=52&length=25)

POST actions belong on single resources or at tenant-level, not on collections. Wiki ref: [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=list%20page%20practices#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=list%20page%20practices#common-issues-found-in-review). This comment applies to all three POST actions being added in the PR (and likely the GET added above as well).

Reply:
fixed